packages/fxa-settings/src/pages/CompleteResetPassword/index.tsx

```diff
@@ -0,0 +1,188 @@
+import { useAccount, useAlertBar } from '../../models';
+import WarningMessage from '../../components/WarningMessage';
+import FormPassword from '../../components/Settings/FormPassword';
+import LinkRememberPassword from '../../components/LinkRememberPassword';
+
+export type CompleteResetPasswordProps = {
+  email?: string;
+  forceEmail?: string;
+  isLinkExpired?: boolean;
+  isLinkDamaged?: boolean;
+  showSyncWarning?: boolean;
+  showAccountRecoveryInfo?: boolean;
+};
+
+type FormData = {
+  newPassword: string;
+  confirmPassword: string;
+};
+
+const CompleteResetPassword = ({
+  email,
+  forceEmail,
+  isLinkExpired,
+  isLinkDamaged,
+  showSyncWarning,
+  showAccountRecoveryInfo,
+}: CompleteResetPasswordProps & RouteComponentProps) => {
+  usePageViewEvent('complete-reset-password', {
+    entrypoint_variation: 'react',
+  });
+
+  const resendLinkHandler = () => {
+    //   TODO: add resend link action
+  };
+
+  const { handleSubmit, register, getValues, errors, formState, trigger } =
+    useForm<FormData>({
+      mode: 'onTouched',
+      criteriaMode: 'all',
+      defaultValues: {
+        newPassword: '',
+        confirmPassword: '',
+      },
+    });
+  const [newPasswordErrorText, setNewPasswordErrorText] = useState<string>();
+
+  const alertBar = useAlertBar();
+  const account = useAccount();
+  const navigate = useNavigate();
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const alertSuccessAndGoHome = useCallback(() => {
+    const successCompletePwdReset = ftlMsgResolver.getMsg(
+      'complete-reset-password-success-alert',
+      'Password set'
+    );
+    alertBar.success(successCompletePwdReset);
+    navigate(HomePath + '#password', { replace: true });
+  }, [alertBar, ftlMsgResolver, navigate]);
+
+  const onFormSubmit = useCallback(
+    async ({ newPassword }: FormData) => {
+      try {
+        // TODO: add logic to set a new password
+        alertSuccessAndGoHome();
+      } catch (e) {
+        // TODO metrics event for error
+        const errorCompletePwdReset = ftlMsgResolver.getMsg(
+          'complete-reset-password-error-alert',
+          'Sorry, there was a problem setting your password'
+        );
+        alertBar.error(errorCompletePwdReset);
+      }
+    },
+    [ftlMsgResolver, alertSuccessAndGoHome, alertBar]
+  );
+
+  return (
+    <>
+      {isLinkExpired && (
+        <>
+          <FtlMsg id="complete-reset-pw-link-expired-header">
+            <h1 id="fxa-reset-link-expired-header" className="card-header">
+              Reset password link expired
+            </h1>
+          </FtlMsg>
+
+          <FtlMsg id="complete-reset-pw-link-expired-message">
+            <p className="mt-4">
+              The link you clicked to reset your password is expired.
+            </p>
+          </FtlMsg>
+          <FtlMsg id="complete-reset-pw-resend-link">
+            <button
+              onClick={resendLinkHandler}
+              className="cta-primary cta-base-p mt-4"
```

Comment:
I think some of these states could use `text-sm` on the paragraph tags. 

Kind of wondering if we should have a 1-2 pointer issue to set `p { @apply text-sm; }` since `text-sm` really seems to be our base for paragraphs, but I'd have to see how that'd affect settings too.